modules/@ergonode/attributes/src/extends/components/Forms/AttributeFormParamsSelect.vue

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <Select
+        :data-cy="dataCyGenerator('params')"
+        key="attrHasParams"
+        :value="value"
+        required
+        :label="paramsLabel"
+        :options="attributeParametersOptions"
+        :error-messages="errorMessages[paramsFieldKey]"
+        :disabled="!isAllowedToUpdate"
+        @input="setParameterValue" />
+</template>
+
+<script>
+import PRIVILEGES from '@Attributes/config/privileges';
+import {
+    toCapitalize,
+} from '@Core/models/stringWrapper';
+import {
+    mapActions,
+    mapState,
+} from 'vuex';
+
+export default {
+    name: 'AttributeFormParamsSelect',
+    components: {
+        Select: () => import('@Core/components/Inputs/Select/Select'),
+    },
+    props: {
+        errorMessages: {
+            type: Object,
+            default: () => ({}),
+        },
+        typeKey: {
+            type: String,
+            default: '',
+        },
+        getParams: {
+            type: Function,
+            default: () => ({}),
+        },
+    },
+    data() {
+        return {
+            parameterData: {},
+        };
+    },
+    computed: {
+        ...mapState('attribute', {
+            attrID: state => state.id,
```

Comment:
keeping it as 'id' variable should be enough - we are in attribute module, also it's not used